content/en/security/cloud_security_management/agentless_scanning/deployment_methods.md

```diff
@@ -47,6 +49,16 @@ The following diagram illustrates how Agentless scanning works when deployed wit
 {{% /tab %}}
 {{< /tabs >}}
 
+## Recommended configuration
+
+To improve the scanner's efficacy, ensure your setup follows those guidelines:
+
+- Scanners are deployed in a single AWS account
+- A scanner is deployed in each region that has either:
+  - More than 250 hosts
+  - Any data store if using [Cloud Storage Scanning](/security/cloud_security_management/agentless_scanning#cloud-storage-scanning)
```

Comment:
```suggestion
- Deploy scanners within a single AWS account
- Deploy a scanner in each region that has more than 250 hosts
- Deploy a scanner in any region containing a data store if using [Cloud Storage Scanning][2]
```

Reply:
Can you add the link to the bottom of the page [2]: /security/cloud_security_management/agentless_scanning#cloud-storage-scanning